sql-statements/sql-statement-create-table.md

```diff
@@ -191,7 +191,7 @@ mysql> DESC t1;
 * 不支持 `FULLTEXT`，`HASH` 和 `SPATIAL` 索引。
 * 为了与 MySQL 兼容，`index_col_name` 属性支持 length 选项，最大长度默认限制为 3072 字节。此长度限制可以通过配置项 `max-index-length` 更改，具体请参阅 [TiDB 配置文件描述](/tidb-configuration-file.md#max-index-length)。
 * 为了与 MySQL 兼容，TiDB 会解析但忽略 `index_col_name` 属性的 `[ASC | DESC]` 索引排序选项。
-* `COMMENT` 属性最多支持 1024 个字符，不支持 `WITH PARSER` 选项。
+* 表的 `COMMENT` 属性最多支持 2048 个字符；列的 `COMMENT` 属性最多支持 1024 个字符，不支持 `WITH PARSER` 选项。
```

Comment:
What about the index comment and table partition comment?
I think it's fine to remove the declaration because it now keeps the same with MySQL.

Reply:
In MySQL the comments' length has two value:

- `1024` for index, column, and table partition
- `2048` for table

In TiDB, I don't see the implementation for table partition comment. The other three comments are the same as MySQL.

Reply:
PTAL